### ISSUE TITLE
Add a view decorator

### ISSUE THREAD

**Issue by ziima** [fabiocaccamo/django-maintenance-mode#23]
For a complex applications, such as we have, the split of views available and unavailable during the maintenance can be non-trivial. Decorator which marks views turned off during the maintenance would allow fine-tuning of the application maintenance.

**Comment by fabiocaccamo** [comment]
This is a very useful feature. Thanks for the suggestion, I will work on it as soon as possible.

**Comment by ziima** [comment]
If you don't persist on writing it yourself, I should be able to find time to do it next week.

**Comment by fabiocaccamo** [comment]
If you can write it with tests it would be great.

By default all views are affected by maintenance-mode when active, so my idea is to add a single decorator named `ignore_maintenance_mode`.

**Comment by ziima** [comment]
That's only true if middleware is turned on. Some of our use cases requires only few views to be turned on, while most of the application is not affected by the middleware. My original idea was to add possitive decorator.

But I see you case, so I suggest to add both decorators - force and ignore, similarly to CSRF in Django itself.

**Comment by fabiocaccamo** [comment]
So you need to have some views under a forced maintenance-mode also if the maintenance-mode is not set to True, is it correct?

**Comment by ziima** [comment]
Nope, we need only a few views to be subject of the maintenance mode, while most of them is not affected. Thus we want to decorate those view and avoid usage of the middleware.

**Comment by fabiocaccamo** [comment]
> By default all views are affected by maintenance-mode when active, so my idea is to add a single decorator named `ignore_maintenance_mode` **(to avoid usage of the middleware for the views that must not be affected by maintenance-mode)**

I think we are saying the same thing...

**Comment by fabiocaccamo** [comment]
I think that the best approach would be: 
```python
MAINTENANCE_MODE_IGNORE_VIEWS = ('app.views.no_maintenance_mode', )
```
Have you any reason to prefere a view decorator?

**Comment by ziima** [comment]
1. I can't imagine this to work for class based views.
2. The list could be really long. In such case, the decorator would be much easier to maintain.

**Comment by fabiocaccamo** [comment]
Released `0.9.0`
Now you can use `force_maintenance_mode_on` and `force_maintenance_mode_off` [decorators](https://github.com/fabiocaccamo/django-maintenance-mode/blob/master/README.md#views).
